sample.json

```diff
@@ -29,6 +29,7 @@
     {
       "company": "Google",
       "type": "Full-time",
+      "icon": "",
```

Comment:
Thanks for this idea to have an icon for the Experience component. But as I said (https://github.com/MehdiKhoshnevisz/resume/pull/1/files#r1057174122) We should have a feature to create a new template. I want to keep the default template simple.